src/main/java/com/devcycle/sdk/server/cloud/api/DVCCloudApiClient.java

```diff
@@ -29,7 +29,13 @@ private DVCCloudApiClient(DVCCloudOptions options) {
     OBJECT_MAPPER.setSerializationInclusion(JsonInclude.Include.NON_NULL);
     okBuilder = new OkHttpClient.Builder();
 
-    url = checkIfStringNullOrEmpty(bucketingApiUrl) ? BUCKETING_URL : bucketingApiUrl;
+    if (!isStringNullOrEmpty(bucketingApiUrl)) {
+      url = bucketingApiUrl;
+    } else if (!isStringNullOrEmpty(options.getBaseURLOverride())) {
```

Comment:
could this not be the first `if` statement and the else be setting it to `bucketingApiUrl`?

Reply:
hmm I think I just wrote this wrong, sec